astroid/tests/unittest_inference.py

```diff
@@ -4273,7 +4291,6 @@ def test_duplicated_keyword_arguments(self):
         self.assertIn('f', site.duplicated_keywords)
 
 
-@unittest.expectedFailure
```

Comment:
Why was this removed? I'm not sure we fixed this.

Reply:
I was trying to stop TravisCI from failing. I'll drop that line and re-push